index.bs

```diff
@@ -201,13 +201,16 @@ The request has an <dfn>associated element</dfn>, the {{Element}} causing this i
 
 NOTE: we assume that there is one <a>image request</a> for each {{Element}} that a <a>background-image</a> property affects and for each URL that the <a>background-image</a> property specifies. So, for example, if there is a style with two URLs affecting all <{div}>s, and there are two <{div}>s, then there will be four <a>image requests</a>. This means that a single <a>background-image</a> property could produce multiple {{PerformanceElementTiming}} entries because it can affect multiple elements and because it can specify multiple URLs.
 
+Every image resource that is obtained from a data URL [[RFC2397]] has an associated <a>image request</a> which is not fetched but still needs to be loaded and decoded.
```

Comment:
I think that falls out from HTML's "update the image data" and Fetch, so you may want to just add a note on that front.
/cc @annevk for a sanity check

Reply:
Hmm yea I'm not sure how data URIs are handled, hence why I added this...

Reply:
yeah, I think you can just turn the above to a note, and change the data URI definition to point to [URL scheme](https://url.spec.whatwg.org/#concept-url-scheme) being "data"